project/ReactiveMongo.scala

```diff
@@ -902,5 +902,5 @@ object Version {
       libraryDependencies ++= Seq(specs) ++ logApi,
       pomPostProcess := providedInternalDeps
     ).enablePlugins(CopyPasteDetector).
-    dependsOn(driver)
+    dependsOn(driver % "test->test;compile->compile")
```

Comment:
That's sharing all the tests classes for one `val`

Reply:
The only way I know to avoid it is to duplicate all the logic. Please suggest an alternative.

Reply:
Duplicate is not always evil ...